tests/e2e/e2e_test.go

```diff
@@ -1,14 +1,67 @@
 package e2e
 
 import (
-	"github.com/osmosis-labs/osmosis/v7/tests/e2e/chain"
+	"fmt"
+	"strconv"
+	"time"
 )
 
-func (s *IntegrationTestSuite) TestIBCTokenTransfer() {
+func (s *IntegrationTestSuite) TestSuperfluidVoting() {
 	chainA := s.chainConfigs[0].chain
-	chainB := s.chainConfigs[1].chain
+	s.Run("superfluid_vote_chainA", func() {
```

Comment:
`s.Run(...)` is normally used for table drive tests to iterate over several test cases and create a runner for each. Here, we already have the test runner created with `TestSuperfluidVoting`, the name also makes sense so I think `s.Run(...)` is not needed